packages/block-editor/src/components/inserter/block-patterns-explorer/pattern-explorer-sidebar.js

```diff
@@ -22,6 +22,7 @@ function PatternCategoriesList( {
 						onClick={ () => {
 							onClickCategory( name );
 						} }
+						tabIndex={ 0 }
```

Comment:
Question: Should we maybe just add `tabIndex={0}` to the `<Button>` component to hide this workaround in an abstraction? IIUC, this doesn't impact other browsers and is actually the spec-compliant. Does adding this introduce any side-effect? Obviously we can do this in a separate PR too.